packaged/lisk.sh

```diff
@@ -108,38 +108,42 @@ create_database() {
 }
 
 populate_database() {
-	if psql -ltAq | grep -q "^$DB_NAME|" >> "$SH_LOG_FILE" 2>&1; then
+	if ! psql -ltAq | grep -q "^$DB_NAME|" >> "$SH_LOG_FILE" 2>&1; then
+		echo "Could not find database $DB_NAME."
+		exit 1
+	fi
+	frobnicate
+	if [ "$DB_DOWNLOAD" = "Y" ]; then
 		download_blockchain
-		restore_blockchain
 	fi
+	restore_blockchain
 }
 
-download_blockchain() {
-	if [ "$DB_DOWNLOAD" = "Y" ]; then
-		rm -f "$DB_SNAPSHOT"
-		if [ "$BLOCKCHAIN_URL" = "" ]; then
-			# performing network check here
-			# if user has not defined snapshot location, only download blockchain for networks where we have a snapshot
-			if [ "$NETWORK" != "main" ] && [ "$NETWORK" != "test" ] && [ "$NETWORK" != "beta" ]; then
-				DB_SNAPSHOT="$MINIMAL_DB_SNAPSHOT"
-				echo "Warning: no snapshot available for $LISK_NETWORK. Syncing from genesis"
-				return
-			fi
+frobnicate() {
```

Comment:
Tempted to go with squash, if only for the great use of The Jargon File http://catb.org/jargon/html/F/frobnicate.html